File: Announcement.Core/Models/SocialUser.cs
using System;
using Newtonsoft.Json;

namespace Announcement.Core
{
    public class SocialUser
    {
        [JsonProperty("userId")]
        public string UserId { get; set; }

		[JsonProperty("displayName")]
		public string DisplayName { get; set; }

        [JsonProperty("token")]
        public String Token { get; set; }
    }
}

File: API/Java/api-stop-spam-LoginViaSocial/src/main/java/api/SocialUser.java
package api;


public class SocialUser 
{
    private String userId;
    
    private String token;
    
    private String displayName;

    public String getUserId() 
    {
        return this.userId;
    }

    public void setUserId(String userId) 
    {
        this.userId = userId;
    }
    
    public String getToken() 
    {
        return this.token;
    }

    public void setToken(String token) 
    {
        this.token = token;
    }

    public String getDisplayName() 
    {
        return this.displayName;
    }

    public void setDisplayName(String displayName) 
    {
        this.displayName = displayName;
    }
}